srml/fees/src/lib.rs

```diff
@@ -33,12 +33,34 @@ mod tests;
 
 type AssetOf<T> = <<T as Trait>::TransferAsset as ArithmeticType>::Type;
 
+pub trait OnFeeCharged<Amount> {
+	fn on_fee_charged(fee: &Amount);
+}
+
+impl<Amount> OnFeeCharged<Amount> for () {
+	fn on_fee_charged(_: &Amount) {}
+}
+
+impl<
```

Comment:
Could you do this implementation please like seen [here](https://github.com/paritytech/substrate/blob/9112a28af677b75cd6f935dda34d283bb4f676e7/srml/session/src/lib.rs#L51).